projects/golang/build.sh

```diff
@@ -13,23 +13,23 @@
 # limitations under the License.
 
 function compile_fuzzer {
-    fuzzer=$(basename $1)
+  fuzzer=$(basename $1)
 
-    # Instrument all Go files relevant to this fuzzer, compile and store in $fuzzer.a
-    go-fuzz -o $fuzzer.a github.com/dvyukov/go-fuzz-corpus/$fuzzer
+  # Compile and instrument all Go files relevant to this fuzz target.
+  go-fuzz -o $fuzzer.a github.com/dvyukov/go-fuzz-corpus/$fuzzer
 
-    # Instrumented, compiled Go ($fuzzer.a) + libFuzzer = fuzzer binary
-    $CXX $CXXFLAGS $LIB_FUZZING_ENGINE $fuzzer.a -lpthread -o fuzzer-$fuzzer
+  # Link Go code ($fuzzer.a) with fuzzing engine to produce fuzz target binary.
+  $CXX $CXXFLAGS $LIB_FUZZING_ENGINE $fuzzer.a -o fuzzer-$fuzzer
 
-    # Copy the fuzzer binary
-    cp fuzzer-$fuzzer $OUT
+  # Copy the fuzzer binary
+  cp fuzzer-$fuzzer $OUT
```

Comment:
nit: i think these two cps can be removed by adding $OUT/ as prefix in the two commands.

Reply:
done!